src/odemis/gui/comp/stream.py

```diff
@@ -2154,11 +2154,48 @@ def __init__(self, parent,
         """
         wx.Panel.__init__(self, parent, wid, pos, size, style, name)
         self.buttons = {}  # int --> wx.Button
+        # Dwell time slider variables
+        self.dwell_time_slider_conf = None
+        self.dwell_time_slider_ctrl = None
+        self._dwell_time_grid_sz = wx.GridBagSizer()
+        self._dwell_time_grid_sz.SetEmptyCellSize((0, 0))
 
         self._panel_sizer = wx.BoxSizer(wx.VERTICAL)
         self.SetSizer(self._panel_sizer)
 
-    def create_controls(self, layout):
+    def _add_dwell_time_slider(self, label_text: str = "Dwell time", value: float = None, conf: dict = None):
+        """ Add a dwell time float value slider to the overview acquisition panel
+
+        :param label_text: (str) Label text to display
+        :param value: (None or float) Value to display
+        :param conf: (None or dict) Dictionary containing parameters for the control
+
+        """
+        # Create label
+        lbl_ctrl = wx.StaticText(self, -1, str(label_text))
+        self._dwell_time_grid_sz.Add(lbl_ctrl, (0, 0),
+                                     flag=wx.ALL | wx.ALIGN_CENTER_VERTICAL, border=5)
+        value_ctrl = UnitFloatSlider(self, value=value, **conf)
+        self._dwell_time_grid_sz.Add(value_ctrl, (0, 1),
+                                     flag=wx.ALL | wx.EXPAND | wx.ALIGN_CENTER_VERTICAL, border=5)
+        return value_ctrl
+
+    def create_controls(self, layout, add_dwell_time_slider: bool = False):
```

Comment:
Instead of getting a bool and hoping that the called has set the `dwell_time_slider_conf` beforehand is confusing. Drop the bool, and directly take the conf as argument.